test/nconf_test.erl

```diff
@@ -14,7 +14,7 @@
 -define(TEST_FILE, "nconf_config_test_file.nconf").
 -define(TEST_FILE_CONTENTS, <<"{a, 1}.\n{b, 2}.">>).
 
--define(assertEqualSorted(A, B), ?assertEqual(lists:sort(A), lists:sort(B))).
+-define(ASSERT_EQUAL_SORTED(A, B), ?assertEqual(lists:sort(A), lists:sort(B))).
```

Comment:
To preserve eunit style, please leave this macro in camelCase and disable elvis rule for this module.